{"update":{"13.10":[{"version":"1:2013.2.3-0ubuntu1.1","package":"python-glance"}]},"special":"The problem can be corrected by updating your system to the following package versions:\nIn general, a standard system update will make all the necessary changes.","date":"May 05, 2014","threat":["A remote authenticated attacker exploit this to execute arbitrary commands as the glance user.."],"usn":"USN-2193-1","CVEs":["CVE-2014-0162"],"info":"glance vulnerability"}